src/btree/bt_ret.c

```diff
@@ -175,14 +183,7 @@ __wt_value_return_upd(
              * room on the stack to avoid memory allocation in normal cases, but we have to handle
              * the edge cases too.
              */
-            if (i >= WT_MODIFY_ARRAY_SIZE) {
-                if (i == WT_MODIFY_ARRAY_SIZE)
-                    listp = NULL;
-                WT_ERR(__wt_realloc_def(session, &allocated_bytes, i + 1, &listp));
-                if (i == WT_MODIFY_ARRAY_SIZE)
-                    memcpy(listp, list, sizeof(list));
-            }
-            listp[i++] = upd;
+            WT_ERR(__wt_modify_vector_push(&modifies, upd));
```

Comment:
The associated comment no longer makes sense.

Reply:
71f63db